<?xml version="1.0"?>
<launch>

    <!-- Launch detection_node -->
    <node pkg="erp42_object_tracker_node" type="erp42_object_tracker_node_detection" name="detection_node" output="screen">
        <rosparam command="load" file="$(find erp42_object_tracker_node)/config/detector_public.yaml"/>
        <rosparam command="load" file="$(find erp42_object_tracker_node)/config/detector_private.yaml"/>
    </node>

    <!-- Launch tracking_node -->
    <node pkg="erp42_object_tracker_node" type="erp42_object_tracker_node_tracking" name="tracking_node" output="screen">
        <rosparam command="load" file="$(find erp42_object_tracker_node)/config/tracker_public.yaml"/>
        <rosparam command="load" file="$(find erp42_object_tracker_node)/config/tracker_private.yaml"/>
    </node>

    <!-- Launch rviz for visualization -->
    <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find erp42_object_tracker_node)/rviz/detection_node.rviz"/> -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find erp42_object_tracker_node)/rviz/detection_node.rviz"/>
</launch>
